packages/editor/src/components/block-icon/index.js

```diff
@@ -18,6 +18,8 @@ function renderIcon( icon ) {
 		}
 
 		return icon();
+	} else if ( icon && icon.type === 'svg' ) {
```

Comment:
I personally don't think it's important enough for people to know precisely that we're adding some props to their SVGs to make them accessible. I guess it doesn't harm to add a mention in the documention but if we do so, we should avoid using `AccessibleSVG` directly in the components because the behavior is documented in that case.

Reply:
That's fair. I think I'd prefer the documentation because it teaches people about an accessibility issue and it means our code can be: 1. a bit leaner (no imports) 2. more idiomatic (regular SVGs)

So maybe just a note of "we add accessibility properties to SVGs" in the block property documentation with a link to the original issue for context or something.